docs/release-notes.txt

```diff
@@ -5,9 +5,92 @@
 ###############
 
 **************
- Version 0.15
+ Version 0.16
```

Comment:
blocking: can you retain the 0.15 header below for all the 0.15 changes?

Reply:
Oops, of course!